addon/components/img-lazy.js

```diff
@@ -50,34 +58,42 @@ export default Component.extend({
   alt: null,
   width: null,
   height: null,
+  srcset: null,
+  sizes: null,
 
   attributeBindings: [
     '_src:src',
+    '_srcset:srcset',
     'title',
     'alt',
     'width',
     'height',
+    'sizes'
   ],
 
   didInsertElement() {
-    get(this, 'observer').observe(this)
+    this._super(...arguments)
```

Comment:
Just for my curiosity, as I see this is done in the official guides too, what are the implications of not calling super in component hooks? Did something change? Were these always necessary and a lot of my components have bugs? Or is this "just" best practice?

Reply:
This is just a best practice. It ensures the chain of `didInsertElement()` calls when you extend a component and implement your own `didInsertElement()`.